### PR TITLE
Feature/security scanning

### PR DESCRIPTION
**Pull request by yinan-symphony**
### Description
Closes #[ISSUE NUMBER]

Please put here the intent of your pull request.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [ ] Referenced an issue in the PR title or description
- [ ] Filled properly the description and dependencies, if any
- [ ] Unit/Integration tests updated or added
- [ ] Javadoc added or updated
- [ ] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java [hunk 1]
```diff
@@ -59,6 +59,7 @@ public static String getTextContent(String presentationML) throws PresentationML
   @SneakyThrows
   private static DocumentBuilder initBuilder() {
     final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+    factory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
```
Comment (symphony-soufiane) [inline symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java:62]: why this change?

**Review by symphony-soufiane — APPROVED** [review]

File: symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java (hunk 1 above)
Comment (yinan-symphony) [reply symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java:62]: It s a security issue fix, proposed by the the security scanning action.
